FastBank.Domain/FriendsRelation.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace FastBank.Domain
+{
+    public class FriendsRelation
+    {
+        public FriendsRelation(Guid realationId, Customer friend1, Customer friend2, bool blocked)
+        {
+            RealationId = realationId;
+            Friend1 = friend1;
+            Friend2 = friend2;
+            Blocked = blocked;
+        }
+
+        public Guid RealationId { get; set; }
+        public Customer Friend1 { get; set; }
+        public Customer Friend2 { get; set; }
```

Comment:
Let's refactor and have more than 2 friends

Reply:
It already is implemented with logic which support 2 or more friends.

Reply:
![image](https://github.com/bulgarchik/FastBank/assets/41870483/4a760189-8761-47d7-9b76-63ea9f95a1ac)